ui/tests/test_url_redirects.py

```diff
@@ -0,0 +1,152 @@
+import http
+
+from django.core.urlresolvers import reverse
+
+import pytest
+
+from ui.urls import (
+    TOS_AND_PRIVACY_REDIRECT_LANGUAGES,
+    ARTICLE_REDIRECTS_MAPPING,
+    INTERNATIONAL_TRANSLATION_REDIRECTS_MAPPING
+)
+
+
+UTM_QUERY_PARAMS = '?utm_source=test%12&utm_medium=test&utm_campaign=test%test'
+
+
+def add_utm_query_params(url):
+    return '{}{}'.format(url, UTM_QUERY_PARAMS)
```

Comment:
Can you please name the elements in the string? `{url}{params}.format(url=url, params=UTM_QUERY_PARAMS)`

Reply:
I find that useful in longer strings, but with short ones, it does not add to readability IMHO